tests/common/helpers/platform_api/fan.py

```diff
@@ -0,0 +1,66 @@
+""" This module provides interface to interact with the fan of the DUT
+    via platform API remotely """
+
+import json
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def fan_api(conn, index, name, args=None):
+    if args is None:
+        args = []
+    conn.request('POST', '/platform/chassis/fan/{}/{}'.format(index, name), json.dumps({'args': args}))
+    resp = conn.getresponse()
+    res = json.loads(resp.read())['res']
+    logger.info('Executing fan API: "{}", index: {}, arguments: "{}", result: "{}"'.format(name, index, args, res))
+    return res
+#
+# Methods inherited from DeviceBase class
+#
+
+def get_name(conn, index):
+    return fan_api(conn, index,'get_name')
```

Comment:
Add space after comma (`index, 'get_name'`). Please do this throughout file.

Tip: Until we have an automated linting solution, I also suggest passing all Python code through http://pep8online.com/ and fixing any PEP8 violations it detects. Note that you can ignore "line too long" warnings; just use your best judgement if a line is over 100 chars.

Reply:
There is a pip module autopep8, I used that to format.
Should be good for formatting now onwards